Library/Homebrew/cmd/upgrade.rb

```diff
@@ -104,6 +106,8 @@ def upgrade
       puts formulae_upgrades.join(", ")
     end
 
+    return puts "Dry run: did not upgrade anything." if args.dry_run?
```

Comment:
Instead: changing the `oh1` above to be `oh1 "Would upgrade ...` and then `return` here.